src/Match2Cases.jl

```diff
@@ -56,19 +56,8 @@ function build_state_machine_core(
             succ = successors(code)
             for i in 1:length(succ)
                 push!(work_queue, succ[i])
-                # we will fall through to the true branch if possible and jump to
-                # the false branch.  So we need a label for the false side.
-                i != 1 && ensure_label!(succ[i], state)
             end
```

Comment:
on line 48, `isnothing(code.action)` is more idiomatic Julia

Reply:
@quinnj suggested `=== nothing`.  See https://github.com/JuliaServices/Rematch2.jl/pull/36.  I need to go back and do a cleanup pass for those kinds of things.